docs/src/forc/manifest_reference.md

```diff
@@ -0,0 +1,55 @@
+# Manifest Reference
+
+The `Forc.toml` (the _manifest_ file) is a compulsory file for each package and it is written in [TOML] format. `Forc.toml` consists of the following fields:
+
+* [`[project]`](#the-project-section) — Defines a sway project.
+  * `name` — The name of the project.
+  * `authors` — The authors of the project.
+  * `organization` — The organization of the project.
+  * `license`— The project license.
+  * `entry` — The entry point of the project.
+  * `implicit_std` -  Controls whether provided `std` version (with current `forc` version) will get added as a dependency _implicitly_.
```

Comment:
Add a recommendation that this should be left as default unless you really know what you're doing.